metaflow/datatools/s3op.py

```diff
@@ -73,17 +92,73 @@ def worker(queue, mode):
             if url is None:
                 break
 
-            if mode == 'download':
+            if mode == 'info':
```

Comment:
remind me why we need this `mode == 'info'` in the worker since we have `get_info` below?

Reply:
Apologies, that was supposed to be in the other PR (the one about moving the get_info to the worker). I'll remove it from this one (it's not used here at all). I think I had started working on it and forgot to remove it from this PR.

Reply:
Done